athena-redshift/src/test/java/com/amazonaws/athena/connectors/redshift/integ/RedshiftIntegTest.java

```diff
@@ -189,14 +188,16 @@ private Stack getRedshiftStack()
      */
     private Endpoint getClusterData()
     {
-        AmazonRedshift redshiftClient = AmazonRedshiftClientBuilder.defaultClient();
+        RedshiftClient redshiftClient = RedshiftClient.builder()
+                .region(DefaultAwsRegionProviderChain.builder().build().getRegion())
+                .build();;
```

Comment:
Just use RedshiftClient.create(). It uses the default AWS region provider chain.

Reply:
addressed the changes suggested.